run-bungeecord.sh

```diff
@@ -2,7 +2,9 @@
 
 : ${TYPE:=BUNGEECORD}
 : ${MEMORY:=512m}
-BUNGEE_HOME=/server
+BUNGEE_HOME=./server
+RCON_JAR_VERSION=1.0.0
```

Comment:
A trick I have learned along the way is to do this and then users can always override with a newer version if you release before this image gets updated:
```suggestion
: ${RCON_JAR_VERSION:=1.0.0}
```